Client/Frontend/Widgets/PhotonActionSheetProtocol.swift

```diff
@@ -177,6 +177,23 @@ extension PhotonActionSheetProtocol {
                 _ = self.profile.history.addPinnedTopSite(site).value
             }
         }
+
+        let sendToDevice = PhotonActionSheetItem(title: Strings.AppMenuSendToDeviceTitleString, iconString: "menu-Send-to-Device") { action in
+            guard let bvc = presentableVC as? PresentableVC & InstructionsViewControllerDelegate & ClientViewControllerDelegate else { return }
+            if !self.profile.hasAccount() {
+                let instructionsViewController = InstructionsViewController()
+                instructionsViewController.delegate = bvc
+                let navigationController = UINavigationController(rootViewController: instructionsViewController)
+                bvc.present(navigationController, animated: true, completion: nil)
```

Comment:
Should there be a return here?

Reply:
Yea makes sense. Lets put a return here so a `ClientPickerViewController` is not also created.